functions/import-geojson.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+"use strict";
+
+const HttpStatus = require("http-status-codes");
+const { db } = require("./common/context");
+const Busboy = require("busboy");
+const JSONStream = require("JSONStream");
+
+/**
+ * Read the body of a multipart HTTP POSTed form containing a GeoJson 'file'
+ * and required 'project' id and 'layer' id to the database.
+ */
+async function importGeoJson(req, res) {
+  if (req.method !== "POST") {
+    return res.status(HttpStatus.METHOD_NOT_ALLOWED).end();
+  }
+
+  const busboy = new Busboy({ headers: req.headers });
+
+  // Dictionary used to accumulate form field values, keyed by field name.
+  const params = {};
+
+  // Handle non-file fields in the form. project and layer must appear
+  // before the file for the file handler to work properly.
+  busboy.on("field", (key, val) => {
+    params[key] = val;
+  });
+
+  // This code will process each file uploaded.
+  busboy.on("file", (_field, file, _filename) => {
+    const { project: projectId, layer: layerId } = params;
+    if (!projectId || !layerId) {
```

Comment:
Can we add some logging here indicating what went wrong?

Reply:
We discussed, will address in follow up.